fortitude/src/rule_redirects.rs

```diff
@@ -57,6 +57,10 @@ static REDIRECTS: LazyLock<HashMap<&'static str, &'static str>> = LazyLock::new(
         ("S041", "MOD011"),
         ("S051", "MOD021"),
         ("S021", "C141"),
+        (
+            "deprecated-assumed-size-character",
+            "deprecated-character-syntax",
```

Comment:
Nice! I'm glad we can do this!

Reply:
Oddly, it doesn't raise a redirect warning. I didn't look into it deeply enough to figure out why. Should I try to get that fixed before the merge?

Reply:
Hmm, a little suspicious. Does it actually work?

Reply:
I mean, the fix might be best in another PR, but it would be good to check if it's a bug